r_python_graph_code.Rmd

```diff
@@ -0,0 +1,440 @@
+# Plotting graph with R v.s. Python
+
+Qi Meng
+
+```{r, include=FALSE}
+library(reticulate)
+knitr::opts_chunk$set(warning = FALSE, message = FALSE)
+knitr::knit_engines$set(python = reticulate::eng_python)
+```
+
+
+```{r}
+library(reticulate)
+library(tidyverse)
+library(ggridges)
+library(openintro)
+library(GGally)
+library(parcoords)
+library(vcd)
+```
+
+In this document, I'll try to summerize some basic plots that we've learned in the course of `STAT GR5702 Exploratory Data Analysis and Visualization` and try to match the R codes with the corresponding Python codes. The examples provided in the document would be relatively simple. And I will introduce/use only some of the commonly used parameters in the examples. Hope the document can help people get some insights about which python packages/function to use if they want to create the plots. 
+
+The plots that I introduce in this document include `Histogram`, `Boxplot`, `Density Curve`, `Ridgeline Plot`, `QQ Plot`, `Scatter Plot`, `Heatmap`, `Parallel Coordinate Plots` and `Mosaic Plot`.
+
+In this document, I referred to some codes used in the lecture and the problem sets. There are also some python codes that I referred online have been cited at the bottom of each section.
+
+## Histogram
+
+------ 
```

Comment:
Remove all the '------' lines from the .Rmd file

Reply:
'------' lines removed.